src/Inventory/Asset/InventoryNetworkPort.php

```diff
@@ -392,17 +390,24 @@ private function handleUpdates()
                     continue;
                 }
 
-                if (property_exists($data, 'logical_number') && $data->logical_number != $db_lnumber) {
-                    $networkport->update(
-                        Sanitizer::sanitize([
-                            'id'              => $keydb,
-                            'logical_number'  => $data->logical_number
-                        ])
-                    );
+                $criteria = [];
+                foreach (['logical_number', 'ifstatus'] as $k) {
+                    if (property_exists($data, $k) && $data->$k != $dbdata_copy[$k]) {
+                        $criteria[$k] = $data->$k;
+                    }
+                }
+
+                if (count($criteria)) {
+                    $criteria['id'] = $keydb;
+                    $criteria['is_dynamic'] = 1;
```

Comment:
I still don't completely understand the purpose of `is_dynamic`, but this flag prevents getting a ton of unnecessary locked fields.

Reply:
If ```is_dynamic``` input is not set, when updating or when creating.
For GLPI, it is then a human who does the action.
So it adds locks if the object is dynamic (and if the user updates some fields).
To prevent this (from inventory) you need to set ```is_dynamic```